docs/core/extensions/service-discovery.md

```diff
@@ -70,21 +91,23 @@ Here's an example demonstrating how to configure endpoints for the service named
 ```json
 {
   "Services": {
+    "https": {
```

Comment:
This is inverted, it should be Services > catalog > https, like here:
![image](https://github.com/dotnet/docs/assets/203839/103645bd-475a-49ab-9181-da89eaa35bf4)